meta-flex-distro/conf/distro/flex-os.conf

```diff
@@ -348,6 +349,11 @@ PUBLIC_REPOS ?= "meta-sokol-flex ${FORKED_REPOS}"
 
 # Define a location for placing external artifacts to be used by the build
 FLEX_EXTERNAL_ARTIFACTS ?= "${TOPDIR}/flex-external-artifacts"
+
+# Add kernel module development capability to SDK
+TOOLCHAIN_HOST_TASK_EXTRA:append = "${@bb.utils.contains('DISTRO_FEATURES', 'flex-debugging', ' nativesdk-flex nativesdk-bison', '', d)}"
+TOOLCHAIN_TARGET_TASK_EXTRA:append = "${@bb.utils.contains('DISTRO_FEATURES', 'flex-debugging', ' flex-kernel-devsrc', '', d)}"
```

Comment:
I don't have a strong opinion here. To me kernel recipes usually reside in the BSP layers, in this case the common BSP layer which seems fine to me

Reply:
I think what he wanna say that as this is upstream layer so if someone will use this layer with his distro and enable flex-debugging he will get an error for missing flex-kernel-devsrc recipe.

Reply:
Yes, exactly. This makes distro layer dependent on common BSP layer.

In our case, common BSP layer depends on distro layer and we use them together. So, we don't see any issue but this requires that distro and common BSP layers are actually interdependent. IMO, we should keep the dependency unidirectional i.e. common BSP layer depends on distro layer only, not the other way around which would require that we move this recipe to distro layer. So that distro layer is complete in itself.